pkg/vcs/git.go

```diff
@@ -635,3 +635,13 @@ func (git *git) CommitExists(commit string) (bool, error) {
 	}
 	return true, nil
 }
+
+func (git *git) ArchiveHeadCommit(repo, commit string) error {
+	tagName := "tag-" + commit
+	git.git("remote", "add", "syz-gitarchive", repo) // ignore errors on re-adding
+	git.git("tag", tagName)                          // ignore errors on re-tagging
+	if _, err := git.git("push", "syz-gitarchive", tagName); err != nil {
```

Comment:
`git help push` suggests that push can be done with repo URL directly.
If so, we could simplify this a bit by not adding remote.

Reply:
Thanks. I like it.

Reply:
Tested "git push repo_url tag tag-name" locally. Looks good.